frontend/src/routes/Search/Details/KubevirtPluginWrapper.tsx

```diff
@@ -2,18 +2,38 @@
 import React, { useContext, createContext } from 'react'
 import { PluginContext } from '../../../lib/PluginContext'
 import { KubevirtPluginData } from '../../../plugin-extensions/extensions/KubevirtContext'
+import { useAllClusters } from '../../Infrastructure/Clusters/ManagedClusters/components/useAllClusters'
+import { ResourceSearchLink } from './DetailsOverviewPage'
+import { K8sGroupVersionKind } from '@openshift-console/dynamic-plugin-sdk'
 
 // Define a default context
 const DefaultKubevirtPluginContext = createContext<KubevirtPluginData>({} as KubevirtPluginData)
 
-const ResourceLink = () => <div>Hello!!</div>
+const getResourceLinkOverride = (clusterName: string) => {
+  const ResourceLink = (props: { name?: string; groupVersionKind?: K8sGroupVersionKind }) => {
+    const { name = '', groupVersionKind } = props
+    if (!groupVersionKind) {
+      return null
+    }
+    const { version, kind } = groupVersionKind
+    return <ResourceSearchLink cluster={clusterName} apiversion={version} kind={kind} name={name} />
```

Comment:
I was hoping we could create a version that looks just like the one in OpenShift, with the resource icon.